pom.xml

```diff
@@ -98,7 +98,7 @@
         <!-- TOOL Properties -->
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <!-- PROJECT Properties -->
-        <release.version>3.0.1</release.version>
+        <release.version>3.1.0</release.version>
```

Comment:
Would it make sense to have this set to ‘${project.version}’ instead?

Reply:
Sorry, but I don't think, that is possible as this property is used in nightly build output directory at Eclipse.org and some POMs/modules in `bundles` subdirectory.

Reply:
I'm afraid we'll keep forgetting to update this property going forward. What about different solution, ie `osgiversion-maven-plugin` can drop defined parts from a version property (see ie [here](https://github.com/eclipse-ee4j/glassfish-ha-api/blob/master/pom.xml#L174)). Would that be useful for this?

Reply:
...but we need ie `-RC` suffix, right?

Reply:
I strongly agree with "forgetting" comment. I'll try something based on http://www.mojohaus.org/build-helper-maven-plugin/regex-property-mojo.html plugin.
About `-RC` suffix. It depends. In Nightly dir no, in Javadoc headers, in https://github.com/eclipse-ee4j/eclipselink/blob/master/foundation/org.eclipse.persistence.core/src/main/resources/org/eclipse/persistence/version.properties maybe (but there is qualifier).

Reply:
in the `version.properties`, `version` should be full `${project.version}` instead. Same in javadoc. So it is probably just a folder name, which would be good to figure out ...

Reply:
Please check latest changes.

Reply:
yup, that's better. Thanks